gammapy/datasets/map.py

```diff
@@ -2342,47 +2344,154 @@ def to_image(self, name=None):
+            vmax = exp_data.data[exp_data.data > 0].max()
+
+            energy_center = exposure_map.geom.axes["energy_true"].center[index]
+
+            exp_data.plot(
+                ax=ax,
+                cmap=cmap,
+                norm=LogNorm(vmin=vmin, vmax=vmax),
+                add_cbar=True,
+            )
+
+            unit = exposure_map.unit.to_string("latex")
+            cbar = ax.images[-1].colorbar  # Access the colorbar
+            cbar.set_label(f"Exposure [{unit}]")  # Set the formatted label
+
+            if energy_center.value < 1e-2 or energy_center.value > 1e2:
+                title = f"Exposure map at {energy_center:.1e}"
+            elif energy_center.value < 1e-1 or energy_center.value > 1e1:
+                title = f"Exposure map at {energy_center:.1f}"
+            else:
+                title = f"Exposure map at {energy_center:.2f}"
+
+            ax.set_title(title)
+            ax.set_box_aspect(1)
+
+        def plot_exposure_profile(ax, exposure_map):
+            exposure_map.plot(ax=ax, ls="solid", marker=None, xerr=None)
+            # Dynamically format the y-axis label
+            unit = exposure_map.unit.to_string("latex")  # Convert unit to LaTeX format
+            ax.set_ylabel(f"Exposure [{unit}]")  # Set the formatted y-axis label
+            ax.set_title("Exposure (at FoV center)")
+            ax.set_box_aspect(1)
+
+        def plot_containment_radius(ax, psf):
+            psf.plot_containment_radius_vs_energy(ax=ax)
+            ax.legend(fontsize="small")
+            ax.set_title("Containment radius (at FoV center)")
+            ax.set_xscale("log")
+            ax.set_yscale("log")
+            ax.set_box_aspect(1)
+
         def plot_mask(ax, mask, **kwargs):
             if mask is not None:
                 mask.plot_mask(ax=ax, **kwargs)
 
-        fig, axes = plt.subplots(
-            ncols=2,
-            nrows=2,
-            subplot_kw={"projection": self._geom.wcs},
-            figsize=figsize,
-            gridspec_kw={"hspace": 0.25, "wspace": 0.1},
+        # Reduce the datasets to 2D if needed
+        countsmapdata = self.counts.reduce_over_axes()
+        npredmapdata = self.npred().reduce_over_axes()
+
+        # Get the corresponding central pixel SpectrumDataset (exposure, edisp, psf)
+        central_pixel = RectangleSkyRegion(
+            self.counts.geom.center_skydir,
+            width=1.01 * self.counts.geom.pixel_scales[0],
+            height=1.01 * self.counts.geom.pixel_scales[1],
+        )
+        central_spectrum_dataset = self.to_spectrum_dataset(central_pixel)
+
+        # Determine plotting limits
+        vmin = npredmapdata.data.min()
+        vmax = npredmapdata.data.max()
+        # Fallback if the map is entirely zero
+        if vmin == 0.0:
+            vmin = np.max([countsmapdata.data.max() * 0.02, countsmapdata.data.min()])
+        if vmax == 0.0:
+            vmax = countsmapdata.data.max()
```

Comment:
maybe:
```suggestion
        vmin = np.minimum(npredmapdata.data.min(), countsmapdata.data.min())
        vmax = np.maximum(npredmapdata.data.max(), countsmapdata.data.max())
```

Reply:
Actually maybe not ok. If one of them is zero, not sure if it will work with the log-scaled mapping?